pkgs/by-name/na/namida/package.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  git,
+  fetchgit,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "namida";
+  version = "0.5";
```

`namida -V` reports its version as `namida 0.5.0`, `version` should be the full version. Commit messages & PR title should also be changed.
```suggestion
  version = "0.5.0";
```

---

Fixed